package.json

```diff
@@ -22,33 +22,26 @@
   "homepage": "https://github.com/autoricardo/carforyou-api-client-pkg#readme",
   "devDependencies": {
     "@carforyou/eslint-config": "4.0.0",
-    "@pika/pack": "0.5.0",
-    "@pika/plugin-build-node": "0.9.2",
-    "@pika/plugin-build-web": "0.9.2",
-    "@pika/plugin-ts-standard-pkg": "0.9.2",
+    "@rollup/plugin-commonjs": "21.0.1",
+    "@rollup/plugin-typescript": "8.3.0",
     "@types/jest": "27.0.3",
     "@types/node": "16.11.11",
     "bundlesize": "0.18.1",
     "jest": "27.3.1",
     "jest-fetch-mock": "3.0.3",
+    "rollup-plugin-dts": "4.0.1",
+    "rimraf": "3.0.2",
     "semantic-release": "17.4.7",
     "ts-jest": "27.0.7",
     "typescript": "4.5.2"
   },
-  "@pika/pack": {
-    "pipeline": [
-      [
-        "@pika/plugin-ts-standard-pkg",
-        {
-          "tsconfig": "tsconfig.pika.json"
-        }
-      ],
-      [
-        "@pika/plugin-build-web"
-      ],
-      [
-        "@pika/plugin-build-node"
-      ]
-    ]
+  "main": "pkg/cjs/index.js",
+  "module": "pkg/esm/index.js",
+  "types": "pkg/index.d.ts",
+  "files": [
+    "pkg/**/*"
+  ],
```

Comment:
pika was setting these on the generated package.json, they now are manually added to the root